app/models/log_event.rb

```diff
@@ -0,0 +1,6 @@
+class LogEvent < ActiveRecord::Base
+
+  belongs_to :user
```

Comment:
If  `user_id` isn't a foreign key, why does LogEvent belong_to user here? I don't quite follow how you're thinking about the relationship between LogEvent and user.